### ISSUE TITLE
  Input In [12]     import Detectors from detector                      ^ SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by akbarali2019** [MehakArora/Ball_Tracking#2]

**Comment by akbarali2019** [comment]
Input In [12]
    import Detectors from detector
                     ^
SyntaxError: invalid syntax


It is giving the following error even I have all dependend files:




![1](https://user-images.githubusercontent.com/52565814/161901246-a345cfae-881c-4dd1-b0d1-a16812427700.PNG)

![2](https://user-images.githubusercontent.com/52565814/161901270-ca1cd87b-c638-4838-84eb-7876fe24ece4.PNG)

![3](https://user-images.githubusercontent.com/52565814/161901283-2752ca84-d73b-4081-9e60-8ba2b6a2ca11.PNG)

![4](https://user-images.githubusercontent.com/52565814/161901300-9a39e509-d6e5-4049-b718-edd8e31ecee6.PNG)

Please anyone know how to fix this issue?

**Comment by iancarson** [comment]
Yeah. Instead of using import Detectors from detector, use from dectector import Detectors. That will be able to fix it.
